package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-native-confetti-cannon",
-  "version": "1.5.2",
+  "name": "@felipecsl/react-native-confetti-cannon",
```

Comment:
?

Reply:
temporarily uploaded it under my account for testing. we should move to @alza before merging or revert